code/modules/customitems/item_defines.dm

```diff
@@ -326,6 +326,11 @@
 		if(H.head == src)
 			H.slurring = max(3, H.slurring) //always slur
 
+/obj/item/clothing/head/beret/fluff/linda //Epic_Charger: Linda Clark
+	name = "Green beret"
+	desc = "A beret, an artists favorite headwear. This one has two holes cut on the edges."
```

Comment:
"an artists" is grammatically incorrect. If you mean a single artist, then it should be "an artist's", if you mean artists as a group, it should be "the favorite headwear of artists".